test/Microsoft.NET.Build.Tests/GivenThatWeWantToTargetNet471.cs

```diff
@@ -127,14 +159,120 @@ public void It_does_not_include_facades_from_nuget_packages()
             outputDirectory.Should().OnlyHaveFiles(new[] {
                 $"{testProject.Name}.exe",
                 $"{testProject.Name}.pdb",
-
-                //  Remove these two once https://github.com/dotnet/sdk/issues/1647 is fixed
+                
                 "System.Net.Http.dll",
                 "System.IO.Compression.dll",
```

Comment:
I'd suggest adding a comment explaining why these two libraries are included.  My understanding is that these were the only contracts from .NET Standard 1.x that have a higher version than what shipped in .NET 4.7.1.

Reply:
yes, that is why. I'll add a comment.